examples/Kaleidoscope/utilities.jl

```diff
@@ -0,0 +1,11 @@
+# Move to LLVM.jl
+
+function position_before_terminator!(builder::LLVM.Builder, bb::LLVM.BasicBlock)
```

Comment:
This is what I talked about before with inserting an instruction before the terminating instruction of the first block.